### ISSUE TITLE
Missing units: fuel useage

### ISSUE THREAD

**Issue by aleppard** [qudt/qudt-public-repo#648]
There does not appear to be miles per gallon, litres per kilometer etc fuel usage units.

**Comment by steveraysteveray** [comment]
Definitely! Please take a look at our submission guidance [here](https://github.com/qudt/qudt-public-repo/wiki/Unit-Vocabulary-Submission-Guidelines.)
If you have the energy (pun intended), you might include litre-per-km (The URI would be unit:L-PER-KiloM). I would have also suggested the commonly used European litres-per-100km, but I'll have to think about whether we really want unit:L-PER-DecaKiloM. This is the first time I have encountered the need to combine standard prefixes...

**Comment by steveraysteveray** [comment]
Actually, that would be unit:L-PER-HectoKiloM...

**Comment by aleppard** [comment]
Thanks. When I have a moment I'll try and get a PR set up.

**Comment by steveraysteveray** [comment]
By the way, we have discussed and searched HectoKiloMetre, and it does occasionally show up, so we're open to adding unit:L-PER-HectoKiloM
